src/Common/tests/XunitTraitsDiscoverers/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -23,12 +23,14 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
         {
             string issue = traitAttribute.GetConstructorArguments().First().ToString();
```

Comment:
Please add some code which checks if we have at least two arguments, as we don't want any exceptions here. I know at this point it might seem that we don't have any other constructors but if we ever change it this piece of code will start throwing and it should be independent to external forces if possible